devstudio/com.jboss.devstudio.central.discovery/plugin.xml

```diff
@@ -207,7 +207,26 @@ reasonable, reporting issues to these providers as required.</description>
                image32="images/openshift-32.png">
          </icon>
          <overview
-               url="http://tools.jboss.org/">
+               url="http://tools.jboss.org/features/openshift.html">
+         </overview>
+      </connectorDescriptor>
+
+      <connectorDescriptor
+            categoryId="org.jboss.tools.central.discovery.a.web"
+            groupId="org.jboss.tools.central.discovery.a.web.core"
+            description="Provides Tooling for Red Hat's Fabric 8 Launcher."
```

Comment:
certificationId="com.jboss.devstudio.discovery.certification.supported"
is missing